packages/fxa-content-server/app/tests/spec/lib/app-start.js

```diff
@@ -139,6 +148,11 @@ describe('lib/app-start', () => {
     assert.isDefined(appStart._getUniqueUserId());
   });
 
+  it('initializeSurveyTargeter creates targeter', () => {
+    appStart.initializeSurveyTargeter();
+    assert.isDefined(appStart._surveyTargeter);
```

Comment:
We should also check that the correct properties have been set in `appStart._surveyTargeter`.